docs/rules/0146/any.md

```diff
@@ -0,0 +1,55 @@
+---
+rule:
+  aip: 146
+  name: [core, '0146', any]
+  summary: Avoid `google.protobuf.Any` fields.
+permalink: /146/standardized-codes
+redirect_from:
+  - /0146/standardized-codes
+---
+
+# Any
+
+This rule discourages the use of `google.protobuf.Any`, as encouraged in
```

Comment:
Nit: I'd probably choose the words used differently - "discourages X..., as encouraged in Y" sounds like this rule contradicts Y regarding X.

How about:
```
This rules discourages the use of `google.protobuf.Any`, as described in [AIP-146][].
```